### PR TITLE
Add reset config to nodes which support and add cli command

### PR DESCRIPTION
**Pull request by mhines01**
This adds 

kne_cli topology reset <config> <device>

to reset the configuration of the node to a default config

if no devices are provides all nodes in the topology will be attempted
--skip will skip nodes which don't support resetcfg but will continue through all nodes

### CODE REVIEW

File: cmd/topology/topology.go [hunk 1]
```diff
@@ -44,8 +60,64 @@ var (
...
+	}
+)
+
+var (
+	skipReset bool
 )
 
+func resetCfgFn(cmd *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return fmt.Errorf("%s: missing args", cmd.Use)
```
Comment (alexmasi) [inline cmd/topology/topology.go:76]: Won't this cause failure if a user specifies a specific device as well? I think you meant

if len(args) < 1

File: cmd/topology/topology.go [hunk 2]
```diff
@@ -44,8 +60,64 @@ var (
...
+	if err != nil {
+		return err
+	}
+	t, err := topo.New(s, topopb)
+	if err != nil {
+		return fmt.Errorf("%s: %w", cmd.Use, err)
+	}
+	ctx := cmd.Context()
+	t.Load(ctx)
+	var nodes []*node.Node
```
Comment (alexmasi) [inline cmd/topology/topology.go:92]: I think this should simplify this block a bit:

nodes := t.Nodes()
if len(args) > 1 {
    n, err := t.Node(args[1])
    if err != nil {
        return err
    }
    nodes = []*node.Node{n}
}

File: cmd/topology/topology.go [hunk 3]
```diff
@@ -44,8 +60,64 @@ var (
...
+	if len(args) == 1 {
+		nodes = t.Nodes()
+	} else {
+		n, err := t.Node(args[1])
+		if err != nil {
+			return err
+		}
+		nodes = append(nodes, n)
+	}
+	var resetable []node.Reseter
```
Comment (alexmasi) [inline cmd/topology/topology.go:102]: nit: resettable and later in the interface its resetter

**Review by alexmasi — APPROVED** [review]

File: cmd/topology/topology.go (hunk 3 above)
Comment (mhines01) [reply cmd/topology/topology.go]: fixed

**Review by alexmasi — APPROVED** [review]

File: cmd/topology/topology.go (hunk 2 above)
Comment (mhines01) [reply cmd/topology/topology.go]: done

File: cmd/topology/topology.go [hunk 4]
```diff
@@ -44,8 +60,62 @@ var (
...
+	ctx := cmd.Context()
+	t.Load(ctx)
+	nodes := t.Nodes()
+	if len(args) > 1 {
+		n, err := t.Node(args[1])
+		if err != nil {
+			return err
+		}
+		nodes = append(nodes, n)
+	}
```
Comment (alexmasi) [inline cmd/topology/topology.go:99]: This will add the specified node to the entire collection of nodes a second time.

Comment (marcushines) [reply cmd/topology/topology.go:99]: grr this why i didn't want to have that pattern

**Review by shubh90 — APPROVED** [review]

**Pull request merged**